azkaban-db/src/test/java/azkaban/db/AzDBTestUtility.java

```diff
@@ -36,7 +36,7 @@ public static DatabaseOperator initQuartzDB() throws Exception {
 
     public EmbeddedH2BasicDataSource() {
       super();
-      final String url = "jdbc:h2:mem:test";
+      final String url = "jdbc:h2:mem:test;IGNORECASE=TRUE";
```

Comment:
Does H2 DB not use the DB schema definition files?

Reply:
H2 DB has the option of setting case insensitive in the url. It is similar to using a collation with strength PRIMARY.
http://www.h2database.com/html/grammar.html#set_ignorecase